css-overflow-5/Overview.bs

```diff
@@ -515,10 +515,34 @@ Or the non-native rendering, aka ''appearance:none''?
 If the former, we'll obviously need to define the interaction with 'appearance'.
 
 
-<h3 id=focus-order>
-Focus Navigation Order</h3>
+<h3 id=overflow-pseudo-focus-behavior>
+Focus behavior</h3>
 
 The above features generate several focusable pseudo-elements.
+This section defines some of the focus related behaviors.
+
+<h4 id=active-element>
+The active element</h4>
+
+When a ''::scroll-button()'' or ''::scroll-marker'' is focused,
+the <l spec=html>[=documentOrShadowRoot.activeElement=]</l>
```

Comment:
Apologies, I'm never sure how to get these html references to link correctly. I want to link to https://html.spec.whatwg.org/multipage/interaction.html#dom-documentorshadowroot-activeelement

Reply:
```suggestion
the {{documentOrShadowRoot/activeElement}}
```